brother_printer_fwupd/utils.py

```diff
@@ -20,6 +21,14 @@
 except ImportError:
     Gooey = None
 
+try:
+    from typing import Literal
+except ImportError:
+    class _Literal:
+        def __getitem__(self, item):
+            return item
+    Literal = _Literal()  # crutch for Python 3.6
```

Comment:
I think, the oldest Python version, we should support, is 3.8. Older versions are EOL.

```suggestion
from typing import Literal
```

Reply:
Done, popped this 3.6-specific commit.